debian/cloudstack-management.postinst

```diff
@@ -57,7 +57,9 @@ if [ "$1" = configure ]; then
     chgrp cloud ${CONFDIR}/${DBPROPS}
     chown -R cloud:cloud /var/log/cloudstack/management
 
-    ln -s ${CONFDIR}/log4j-cloud.xml ${CONFDIR}/log4j.xml
+    if [ ! -e "$CONFDIR/log4j.xml" ]; then
+        ln -s ${CONFDIR}/log4j-cloud.xml ${CONFDIR}/log4j.xml
```

Comment:
@marcaurele perhaps we can fix it as `ln -sf`?

Reply:
Yep